js/apps/admin-ui/src/user-federation/ldap/LdapSettingsSearching.tsx

```diff
@@ -403,6 +404,38 @@ export const LdapSettingsSearching = ({
             )}
           ></Controller>
         </FormGroup>
+        <FormGroup
+          label={t("referral")}
+          labelIcon={
+            <HelpItem helpText={t("referralHelp")} fieldLabelId="referral" />
+          }
+          fieldId="kc-referral"
+        >
+          <Controller
+            name="config.referral[0]"
+            defaultValue=""
+            control={form.control}
+            render={({ field }) => (
+              <Select
+                toggleId="kc-referral"
+                required
```

Comment:
Is this a required field? There are no annotations on the `FormField` to indicate so.
```suggestion
```

Reply:
No, it is not required. I removed `required`.

Not sure what is the practice, but I did not add empty select option for the menu. Currently the selection defaults to `ignore` and that reflects how it works on the Java side - the JDK LDAP implementation uses `ignore` by default, even when the `java.naming.referral` property is not set at all.

Reply:
That's probably fine, I will consider this resolved.